mitosheet/src/mito/components/Mito.tsx

```diff
@@ -7,25 +7,24 @@ import React, { Fragment, useCallback, useEffect, useRef, useState } from 'react
 
     Except we put the colors.css first because it creates variables used elsewhere.
 */
-import '../../css/sitewide/all-modals.css';
-import '../../css/sitewide/animations.css';
-import '../../css/sitewide/backgrounds.css';
-import '../../css/sitewide/borders.css';
-import '../../css/sitewide/colors.css';
-import '../../css/sitewide/cursor.css';
-import '../../css/sitewide/element-sizes.css';
-import '../../css/sitewide/flexbox.css';
-import '../../css/sitewide/fonts.css';
-import '../../css/sitewide/height.css';
-import '../../css/sitewide/hover.css';
-import '../../css/sitewide/icons.css';
-import '../../css/sitewide/margins.css';
-import '../../css/sitewide/paddings.css';
-import '../../css/sitewide/scroll.css';
-import '../../css/sitewide/text.css';
-import '../../css/sitewide/widths.css';
+import '../../../css/sitewide/all-modals.css';
```

Comment:
Maybe this file should come out of the `components` subfolder and as a top layer in the `mito` folder